ark/utils/segmentation_utils.py

```diff
@@ -33,6 +35,56 @@ def find_nuclear_mask_id(nuc_segmentation_mask, cell_coords):
     return nuclear_mask_id
 
 
+def split_large_nuclei(cell_segmentation_mask, nuc_segmentation_mask, cell_ids):
+    """Splits nuclei that are bigger than the corresponding cell into multiple pieces
+
+    Args:
+        cell_segmentation_mask (np.ndarray):
```

Comment:
sphinx/rtd might want these types to be `numpy.ndarray` for proper linking to numpy's docs, but double check the docs output on this pr.  no big deal either way, just a heads up.

Reply:
Yes, that's correct. You should change this.